ape_avalanche/ecosystem.py

```diff
@@ -30,11 +50,71 @@ class AvalancheConfig(PluginConfig):
     mainnet_fork: NetworkConfig = _create_local_config()
     fuji: NetworkConfig = _create_network_config()
     fuji_fork: NetworkConfig = _create_local_config()
-    local: NetworkConfig = NetworkConfig(default_provider="test")
+    local: NetworkConfig = _create_local_config(default_provider="test")
     default_network: str = LOCAL_NETWORK_NAME
 
 
 class Avalanche(Ethereum):
     @property
     def config(self) -> AvalancheConfig:  # type: ignore
         return self.config_manager.get_config("avalanche")  # type: ignore
+
+    def create_transaction(self, **kwargs) -> TransactionAPI:
+        """
+        Returns a transaction using the given constructor kwargs.
+        Overridden because does not support
+
+        **kwargs: Kwargs for the transaction class.
+
+        Returns:
+            :class:`~ape.api.transactions.TransactionAPI`
+        """
+
+        transaction_type = self.get_transaction_type(kwargs.get("type"))
+        kwargs["type"] = transaction_type.value
+        txn_class = _get_transaction_cls(transaction_type)
+
+        if "required_confirmations" not in kwargs or kwargs["required_confirmations"] is None:
+            # Attempt to use default required-confirmations from `ape-config.yaml`.
+            required_confirmations = 0
+            active_provider = self.network_manager.active_provider
+            if active_provider:
+                required_confirmations = active_provider.network.required_confirmations
+
+            kwargs["required_confirmations"] = required_confirmations
+
+        if isinstance(kwargs.get("chainId"), str):
+            kwargs["chainId"] = int(kwargs["chainId"], 16)
+
+        if "hash" in kwargs:
+            kwargs["data"] = kwargs.pop("hash")
+
+        if all(field in kwargs for field in ("v", "r", "s")):
+            kwargs["signature"] = TransactionSignature(
+                v=kwargs["v"],
+                r=bytes(kwargs["r"]),
+                s=bytes(kwargs["s"]),
+            )
+
+        return txn_class.parse_obj(kwargs)
+
+    def get_transaction_type(self, _type: Optional[Union[int, str, bytes]]) -> TransactionType:
+        if _type is None:
+            version = TransactionType.STATIC
```

Comment:
wait, do we want the default to be STATIC here? Does Avalanche not support 1559?